client/src/main.jsx

```diff
@@ -123,8 +131,4 @@ const router = createBrowserRouter([
 
 const root = ReactDOM.createRoot(document.getElementById("root"));
 
-root.render(
-  <React.StrictMode>
-    <RouterProvider router={router} />
-  </React.StrictMode>
-);
+root.render(<RouterProvider router={router} />);
```

Comment:
wasn't it working yesterday with strict mode with the last changes you implemented? (sorry i'm trying to keep track)